monkey/monkey_island/cc/services/attack/technique_reports/__init__.py

```diff
@@ -60,6 +60,15 @@ def technique_status(technique):
         else:
             return ScanStatus.UNSCANNED
 
+    @staticmethod
+    def get_message_and_status(technique, status):
```

Comment:
create a function `get_message_by_status` which just returns the corresponding message without wrapping it in a dictionary with status.
You can keep this (`get_message_and_status`) function, but simplify it using `get_message_by_status`

Reply:
smart